test/test-server/org_handlers.go

```diff
@@ -57,17 +60,51 @@ func handleOrgEnvironments(t *testing.T) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
 		if r.Method == http.MethodGet {
-			environmentList := &orgv2.OrgV2EnvironmentList{Data: getOrgEnvironmentsList(OrgEnvironments)}
+			environmentList := &orgv2.OrgV2EnvironmentList{Data: getOrgResourcesList[orgv2.OrgV2Environment](OrgEnvironments)}
 			err := json.NewEncoder(w).Encode(environmentList)
 			require.NoError(t, err)
 		}
 	}
 }
 
-func getOrgEnvironmentsList(envs []*orgv2.OrgV2Environment) []orgv2.OrgV2Environment {
-	envList := []orgv2.OrgV2Environment{}
-	for _, env := range envs {
-		envList = append(envList, *env)
+// Handler for: "/org/v2/organizations/{id}"
+func handleOrgOrganization(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		orgId := vars["id"]
+		w.Header().Set("Content-Type", "application/json")
+
+		if r.Method == http.MethodGet {
+			if orgId == "abc-789" {
+				w.WriteHeader(http.StatusForbidden)
+			} else {
+				organization := &orgv2.OrgV2Organization{
+					Id:          orgv2.PtrString(orgId),
+					DisplayName: orgv2.PtrString("default"),
+				}
+				err := json.NewEncoder(w).Encode(organization)
+				require.NoError(t, err)
+			}
+		}
+	}
+}
+
+// Handler for: "/org/v2/organizations"
+func handleOrgOrganizations(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		if r.Method == http.MethodGet {
+			organizationList := &orgv2.OrgV2OrganizationList{Data: getOrgResourcesList[orgv2.OrgV2Organization](OrgOrganizations)}
```

Comment:
This is the only place we're using `OrgOrganizations`. Let's just define it as an `[]*OrgV2Resource` right off the bat. Also- no need for it to be global **or** public as far as I can tell.